### ISSUE TITLE
problem with bucketing lstm and cnn

### ISSUE THREAD

**Issue by xinghedyc** [dmlc/mxnet#4191]
Hi everyone, I want to use lstm and cnn to implement text recognition from image. for example recognize 'good' from a text image bounding box. 
and I use warp-ctc as loss function but I find my softmax out are always the same, like:
    [ 0.01282051,  0.01282051,  0.01282051, ...,  0.01282051,
         0.01282051,  0.01282051],
       [ 0.01282051,  0.01282051,  0.01282051, ...,  0.01282051,
         0.01282051,  0.01282051],
       [ 0.01282051,  0.01282051,  0.01282051, ...,  0.01282051,
         0.01282051,  0.01282051]], dtype=float32))
0.01282051=1/78 as I have 78 different chars to classify.
my dataiter is based on bucket_io.py in rnn example dir, and lstm part is based on warp-ctc example.
I'm confused about this problem , do you have any ideas?  Thank a lot!

**Comment by piiswrong** [comment]
please conform to the issue template

**Comment by xinghedyc** [comment]
sorry, I raise a new one and so close this issue.
